packages/syslog/syslog.2.0.0/opam

```diff
@@ -0,0 +1,31 @@
+opam-version: "2.0"
+
+synopsis: "syslog(3) routines for ocaml"
+
+authors: [
+  "Shawn Wagner"
+  "Eric Stokes"
+]
+
+maintainer: [ "Julien Sagot <julien.sagot@geneanet.org>" ]
+
+homepage: "http://github.com/geneanet/ocaml-syslog"
+dev-repo: "git://github.com/geneanet/ocaml-syslog"
+bug-reports: "https://github.com/geneanet/ocaml-syslog/issues"
+
+build: [ make "build" ]
```

Comment:
Could you use the `dune` commands instead of the build/install/remove fields you have here? Look for other packages using dune to know which command to use (only the `build` field is necessary, the others shouldn't be there)

Reply:
No `install` or `remove` field needed?

Reply:
no

Reply:
dune build already creates an `.install` file that opam uses to install and uninstall all the necessary files

Reply:
Should that `syslog.install` file be versioned in the repo so that opam do not need to build the library for uninstall? Or would it build it anyway?

Reply:
You don't need to version it, it is kept by opam on install and used for the uninstall